worked_examples/geospatial/workshop_april_2024/Workshop_AprilMay2024_DeepSolo.ipynb

```diff
@@ -0,0 +1,361 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mapreader Workshop - 30th April - 1st May 2024"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First check you have the correct version of MapReader. \n",
+    "For the April 2024 workshop, this is v1.3.0\n",
+    "\n",
+    "This can be downloaded from pypi using `pip install mapreader==1.3.0` or by checking out the repo at [this commit](https://github.com/Living-with-machines/MapReader/releases/tag/v1.3.0)"
```

Comment:
We tell people elsewhere to install from source. Is this OK to have here as different instruction?